docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: page
+title: Alluxio
+nav_order: 5
+parent: Getting-Started
+---
+# Get Started with RAPIDS with Alluxio
+
+Alluxio is a data orchestration platform brings your data closer to compute across clusters,
+regions, clouds, and countries. This guide will go through how to set up the RAPIDS Accelerator
+for Apache Spark 3.0 with Alluxio and on premise cluster.
+
```

Comment:
we may want to add a bit more why you want to use alluxio and perhaps also that performance varies based on hardware - ssd, spinning disk, etc.

Reply:
Done